desk/lib/bolt.hoon

```diff
@@ -6,24 +6,23 @@
 ::  ...
 ::  %-(agent:bolt your-agent)
 ::
-
 /-  *bolt
 /+  verb, agentio
-
+::
 =>
-|% 
-+$  card  card:agent:gall
-+$  versioned-state
-  $%  state-0
-  ==
-+$  state-0  
-  $:  %0 
-      which=?
-      kids=?
-      =blacklist 
-      =whitelist
-  ==
---
+  |% 
+  +$  card  card:agent:gall
+  +$  versioned-state
+    $%  state-0
+    ==
+  +$  state-0  
+    $:  %0 
+        white=?
+        kids=?
+        blacklist=(set ship) 
+        whitelist=(set ship)
```

Comment:
Previously these were separate types, but since there is no difference between them (and we never use the atom that you tagged them with for anything), I would suggest disambiguating them purely based on their names instead. This makes it possible to write `(~(has in blacklist) user)` instead of having to do `users.blacklist`, and is just generally more direct, less bureaucratic.